### PR TITLE
74399 ARF - Rename to arp and representative

### PR DESCRIPTION
**Pull request by williamphelps13**
## Summary
- Purpose
  - There are a number of places we are using `reps` or `representatives` such as the Github vets-website directory name, in our manifest, etc
  - As a team we decided the subdomain should be representative.va.gov (6 to 2 with 2 beings reps.va.gov) and as such uses of `reps` or `representatives` (plural) should be renamed to `rep` or `representative`
  - It was decided as a team that `rep` can be used in instances in which brevity is important
  - In addition, the portal within this subdomain now has a name - Accredited Representative Portal. As such, there are cases where `representatives` should actually be changed to `accredited-representative-portal` 
- Changes that have been made to the platform:
  - Update feature flag name to `accreditedRepresentativePortalFrontend`
  - Update feature flag name where used in App and in e2e tests
  - Rename styling file to `representative.scss`
  - Update all other uses of `representatives` to `representative` EXCEPT the path because then content build would not build
  - Update some uses of `representative to instead by the more specific `Accredited Representative Portal`
- Team: Benefits Accredited Representative Facing Team

## Related issue

[#74399 Rename to representative / accredited-representative-portal](https://app.zenhub.com/workspaces/accredited-representative-facing-team-65453a97a9cc36069a2ad1d6/issues/gh/department-of-veterans-affairs/va.gov-team/74399)

## Related PRs
- vets-api toggle update: [74399 ARF - Rename arp feature toggle #15876](https://github.com/department-of-veterans-affairs/vets-api/pull/15876)
## Testing done

- Steps required to verify the changes are working as expected: Manually click through all functionality of the application including -
  - Landing page Link links to Dashboard
  - Dashboard POA Requests Widget has data and can be sorted
  - Dashboard Sidenav links to POA Requests page
  - POA Requests Sidenav links to Dashboard
  - Dashboard `View all` links to POA Requests page
  - POA Requests Table has data and can be sorted
  - Clicking Accept/Decline results in currently erroring but correctly pointing API calls in the console
  - POA Requests Sidenav links to Permissions Page
  - Home Breadcrumb navigates back to Landing Page

## Screenshots
- No visual UI changes

## What areas of the site does it impact?
- Only impacts the application /representatives and related config and feature toggle files

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### CODE REVIEW

File: src/applications/representatives/manifest.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "appName": "Representatives",
+  "appName": "Accredited Representative Portal",
   "entryFile": "./app-entry.jsx",
-  "entryName": "representatives",
-  "rootUrl": "/representatives",
+  "entryName": "representative",
+  "rootUrl": "/representative",
```
Comment (nihil2501) [inline src/applications/representatives/manifest.json:5]: Root url = `/accredited-representative-portal`?

File: src/applications/representatives/tests/components/common/Breadcrumbs.unit.spec.js [hunk 2]
```diff
@@ -7,30 +7,30 @@ import Breadcrumbs from '../../../components/common/Breadcrumbs';
 describe('breadcrumbs', () => {
   it('creates breadcrumbs for the dashboard', () => {
     const { getByText } = render(
-      <Breadcrumbs pathname="/representatives/dashboard" />,
+      <Breadcrumbs pathname="/representative/dashboard" />,
```
Comment (nihil2501) [inline src/applications/representatives/tests/components/common/Breadcrumbs.unit.spec.js:10]: Root url = `/accredited-representative-portal`?

File: src/platform/user/authentication/constants.js [hunk 3]
```diff
@@ -111,7 +111,7 @@ export const EXTERNAL_REDIRECTS = {
   [EXTERNAL_APPS.EBENEFITS]: `${eAuthURL}/ebenefits`,
   [EXTERNAL_APPS.VA_FLAGSHIP_MOBILE]: '',
   [EXTERNAL_APPS.VA_OCC_MOBILE]: `${eAuthURL}/MAP/users/v2/landing`,
-  [EXTERNAL_APPS.ARP]: `${environment.BASE_URL}/representatives`,
+  [EXTERNAL_APPS.ARP]: `${environment.BASE_URL}/representative`,
```
Comment (nihil2501) [inline src/platform/user/authentication/constants.js:114]: Root url = `/accredited-representative-portal`?

**Review by nihil2501 — COMMENTED** [review]
Root url = `/accredited-representative-portal`?

Even when we have [representative.VA.gov](representative.VA.gov), the easiest thing for us technically that already works will be to do our routing from this redundant base path, so it would be visible to users if we go with that approach. I don't see a huge reason to prefer either [representative.VA.gov/representative](representative.VA.gov/representative) or [representative.VA.gov/accredited-representative-portal](representative.VA.gov/accredited-representative-portal).

**Review by asg5704 — DISMISSED** [review]
Identity code is good ✅

File: src/applications/representatives/manifest.json (hunk 1 above)
Comment (nihil2501) [reply src/applications/representatives/manifest.json:5]: If we can figure out if it's possible, we could change to `"rootUrl": "/"`. Someone would have to think through that. How would you get that to work on localhost and other environments? Here are the [docs around routing](https://depo-platform-documentation.scrollhelp.site/developer-docs/frontend-architecture#Frontendarchitecture-RoutingforReactapps).

**Review by nihil2501 — DISMISSED** [review]
Agree with the principle to move to representative

**Review by cohnjesse — DISMISSED** [review]
LGTM

**Comment by asg5704** [comment]
> Thank you @asg5704 for running that locally! I have [this PR open in content-build](https://github.com/department-of-veterans-affairs/content-build/pull/1957/files) that updates that, and you're saying that that will solve the issue?

Yes that is correct. Once the `content-build` is merged, re-run the E2E tests here and it should be good to go. (I did this locally and they all ran and passed)

**Review by gabezurita — DISMISSED** [review]
LGTM!

**Review by gabezurita — APPROVED** [review]
Excellent job getting CI green on this! That was tough.

LGTM!

**Review by asg5704 — APPROVED** [review]

**Review by jamigibbs — APPROVED** [review]

**Pull request merged**